app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @review = Review.find(params[:review_id])
+    @review.votes.where(:user_id => current_user.id).destroy_all
```

Comment:
What happens if a user isn't signed in? Do you require authentication for this controller?